physfs/physfs.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "PhysicsFS",
+    "buildsystem": "cmake-ninja",
+    "config-opts": [ "-DPHYSFS_BUILD_TEST=OFF" ],
+    "cleanup": [
+        "/lib/*.a",
```

Comment:
Would just `*.a` work?

Reply:
also the config `-DPHYSFS_BUILD_STATIC=OFF` would also do it (better)